### DISCUSSION TITLE
What is the difference between docker versions?

### DISCUSSION THREAD

**Discussion by sofakng** [browserless/chrome#3678] [Q&A] [answered]
What's the difference between the docker versions?

For example, 'latest', '1-chrome-stable', and '1.61-puppeteer-21.4.1' ?

Does the 'latest' tag use v2.0 or v1.xx of Browserless?

**Comment by joelgriffith** [comment, marked as answer]
It depends on where you're getting the image :) V1 is on docker hub, and `latest` represents the bleeding and latest code for V1. V2 is hosted purely here: https://github.com/orgs/browserless/packages
